src/pages/demo/custom-controls-polygon/index.md

```diff
@@ -57,7 +57,7 @@ fabricObject.setPositionByOrigin(absolutePoint, newX + 0.5, newY + 0.5);
 ```
 
 
-### Creating the control
+## Creating the control
```

Comment:
For topics within the content, we'll use **H2** header element instead of **H3**